channelz/service/service_test.go

```diff
@@ -32,10 +32,13 @@ import (
 	channelzpb "google.golang.org/grpc/channelz/grpc_channelz_v1"
 	"google.golang.org/grpc/connectivity"
 	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/grpclog"
 	"google.golang.org/grpc/internal/channelz"
 	"google.golang.org/grpc/internal/grpctest"
 )
 
+var logger = grpclog.Component("channelz")
```

Comment:
Move this to `channelz/service/service.go`, or any other non-test file. You can still use it in the following tests.

Reply:
I made this change. I also fixed the log depths in a slightly different way: by adding +1 to all the depth functions in `grpclog/component.go` which should handle all depth logging done through components.